test/parallel/test-v8-coverage.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+const { spawnSync } = require('child_process');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+let dirc = 0;
+function nextdir() {
+  return `cov_${++dirc}`;
+}
+
+// outputs coverage when event loop is drained, with no async logic.
+{
+  const coverageDirectory = path.join(tmpdir.path, nextdir());
+  const output = spawnSync(process.execPath, [
+    require.resolve('../fixtures/v8-coverage/basic')
+  ], { env: extendEnv({ NODE_V8_COVERAGE: coverageDirectory }) });
```

Comment:
I suggest to use the object spread notation instead of using a helper function here.

```js
{ env: { ...process.env, NODE_V8_COVERAGE: coverageDirectory } }
```